django/contrib/admin/static/admin/css/forms.css

```diff
@@ -82,14 +85,18 @@ form ul.inline li {
     height: 26px;
 }
 
-.aligned label + p, .aligned label + div.help, .aligned label + div.readonly {
+.aligned label + p, .aligned .checbox-row + div.help, .aligned label + div.readonly {
```

Comment:
Typo?

Reply:
Indded yes 😳, fixed now. It also revealed another issue with the datetime widget that I've fixed now.

Hopefully there are not too many more exceptions, this bit of the CSS is quite weird.